apps-rendering/src/components/RelatedContent/GalleryRelatedContent.tsx

```diff
@@ -0,0 +1,104 @@
+// ----- Imports ----- //
+
+import { css } from '@emotion/react';
+import { from, neutral, remSpace, until } from '@guardian/source-foundations';
+import type { Option } from '@guardian/types';
+import { none } from '@guardian/types';
+import BylineCard from 'components/BylineCard';
+import Card from 'components/Card';
+import { grid } from 'grid/grid';
+import type { ResizedRelatedContent } from 'item';
+import { maybeRender } from 'lib';
+import type { FC } from 'react';
+import {
+	COMMENT,
+	defaultHeadingStyles,
+	defaultListStyles,
+} from './RelatedContent.defaults';
+
+// ----- Component ----- //
+
+const styles = css`
+	${grid.container}
+	background-color: ${neutral[97]};
+`;
+
+const relatedContentStyles = css`
+	${grid.column.centre}
+	grid-row: 2;
+	border: 0;
+	position: relative;
+	padding-top: ${remSpace[3]};
+	padding-bottom: ${remSpace[3]};
+
+	${until.wide} {
+		padding-left: 0;
+		padding-right: 0;
+	}
+
+	${from.leftCol} {
+		grid-row: 1;
+		${grid.between('centre-column-start', 'right-column-end')}
+	}
```

Comment:
I think we may want this two-column layout to happen one breakpoint earlier, otherwise the list of content gets cut off with the current desktop designs.

```suggestion
	${from.desktop} {
		${grid.between('centre-column-start', 'right-column-end')}
	}

	${from.leftCol} {
		grid-row: 1;
	}
```